tests/feature/AddressTest.php

```diff
@@ -35,23 +35,25 @@ public function itCanShow()
      */
     public function itCanStore()
     {
+        $addresses = factory(Address::class, 2)->create();
```

Comment:
you dont need to catch these in variable because you are not using them:
`factory(Address::class, 2)->create();`